docs/pages/ui-programming/react-native-toast.md

```diff
@@ -0,0 +1,117 @@
+---
+title: How to display a toast in React Native
```

Comment:
```suggestion
title: How to display a popup toast
```

something like this fits in better with the docs and also hints at what a toast is. also would name this file just toast.md.